web-components/icon/Icon.svelte

```diff
@@ -0,0 +1,74 @@
+<svelte:options tag="leo-icon" />
+
+<script context="module" lang="ts">
```

Comment:
Curiosity more than anything: why is this code in `context="module"` and not the main `script` tag?

Reply:
So that's so the cache is shared between instances of the icon component. If it was in the main script tag each icon would fetch the SVG again.